src/katsdpmodels/band_mask.py

```diff
@@ -0,0 +1,102 @@
+################################################################################
+# Copyright (c) 2020, National Research Foundation (SARAO)
+#
+# Licensed under the BSD 3-Clause License (the "License"); you may not use
+# this file except in compliance with the License. You may obtain a copy
+# of the License at
+#
+#   https://opensource.org/licenses/BSD-3-Clause
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+"""Masks for roll-off at the edges of a band."""
+
+from typing import Type, TypeVar, ClassVar, Any
+from typing_extensions import Literal
+
+import numpy as np
+import astropy.units as u
+import astropy.table
+import h5py
+
+from . import models
+
+_B = TypeVar('_B', bound='BandMaskRanges')
+
+
+class Band:
+    """Defines a band for use with :class:`BandMask`."""
+
+    def __init__(self, bandwidth: u.Quantity, centre_frequency: u.Quantity):
```

Comment:
center_frequency doesn't always guarantee being able to identify the receiver in use, should we include the band description (L,UHF,S) for future proofing...

Reply:
The intent here was just to capture the stuff that is tunable per receiver e.g. narrow-band or S-band mixer. Anything that needs to be handled differently for different receivers is handled as part of selecting the model. I can see how the name could be confusing though - maybe I should call this SpectralWindow instead?

Reply:
Probably a good idea given how overloaded Band is.